blogposts/2020/announcing-sourcegraph-3.16.md

```diff
@@ -0,0 +1,249 @@
+  <p style="text-align: center"><a href="https://youtu.be/{VIDEO_ID}" target="_blank">View on YouTube</a></p>
+</p>
+-->
+
+In Sourcegraph 3.15, we introduced the AND/OR operators on file contents. In Sourcegraph 3.16, we are removing the experimental flag and making this available to all users.
+
+## Performance improvements for precise code intelligence
+
+![performance-improvements-code-intel](/blog/3-16-ts-to-go-lsif.png "performance-improvements-code-intel")
+
+As of Sourcegraph 3.16, our precise code intelligence backend has been rewritten from TypeScript to Go. This effort is part of a larger effort to aggressively optimize conversion and querying of LSIF data, which will start becoming more apparent in upcoming releases. Even with this initial improvement, we’re seeing improvements on precise code intelligence query performance. The following graph shows the five-run average duration of our [integration test](https://github.com/sourcegraph/sourcegraph/tree/5f51043ad2130a1acdcfca8b969f907cd03a220d/internal/cmd/precise-code-intel-test) before and after the rewrite.
+
+## Discover useful scopes with `repogroup` autocompletion
+
+TODO: video
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://www.youtube.com/embed/{VIDEO_ID}" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://youtu.be/{VIDEO_ID}" target="_blank">View on YouTube</a></p>
+</p>
+-->
+
+The `repogroup` filter is useful for creating collections of repositories to search over within Sourcegraph. However, there wasn’t an easy way for users to learn what `repogroups` were available. Now, Sourcegraph provides autocompletion to help you discover available options. This happens automatically when you start typing, or can be triggered using the ctrl-spacebar keyboard shortcut. Try it out in the new smart search bar for plain text mode or in interactive mode!
+
+## Check syncing status of repository and user permissions
+
+![repository-permissions](https://user-images.githubusercontent.com/2946214/82042090-dd1eaa00-96db-11ea-9fd7-99e0e82a3d0b.png)
+
+Sourcegraph 3.14 and 3.15 introduced a more performant background permissions syncing solution for Bitbucket Server, GitLab, and GitHub repository permissions. Sourcegraph 3.16 adds new permissions pages to user and repository settings when background permissions syncing is enabled. These pages allow admins to check the last time permissions were synced and manually trigger the syncing process for an immediate refresh to the user or repository’s permissions.
+
+Enable this feature by adding ` "permissions.backgroundSync": {"enabled": true}` in your site configuration settings.
+
+## Understanding user activity of your instance
+
+![usage-activity-histogram](/blog/3-16-activity-histogram.png "usage activity histogram")
```

Comment:
@ebrodymoore - the image from the google doc was low quality - can you upload a higher resolution version here or send to me in Slack?